fluent/protocol/message.go

```diff
@@ -21,6 +23,22 @@ type Message struct {
 	Options *MessageOptions
 }
 
+// NewMessage creates a Message from the supplied
+// tag and record. It will set Timestamp to
+// time.Now().UTC().Unix().
+func NewMessage(
+	tag string,
+	record Record,
+) *Message {
+	msg := &Message{
+		Tag:       tag,
+		Timestamp: time.Now().UTC().Unix(),
```

Comment:
is type `int64` for field Timestamp correct? Shouldn't it be `protocol.EventTime`?

This does belong to the `Message` struct, but I can't remark there.